_slides/09-skilambi943.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+this is sailaja--how are you all??
```

Comment:
```suggestion
This is Sailaja -- how are you all?
```

Reply:
Hey, thanks for catching that! Will change

Reply:
very good suggestion!! but i like 2 ??